### PR TITLE
Mobile Control Page updates for build

### PR DESCRIPTION
**Pull request by bigbadcapers**
#### Pull request checklist

- [ ] Added mobile control page markdown
- [ ] Added references to fabric-cdn to use images hosted there
- [ ] Took merges from https://github.com/ecraig12345/office-ui-fabric-react/tree/more-controls **please verify I didnt mess anything up here 😄 **

- [ ] I still dont know how to avoid all these other branches' commits and edits from polluting the PR. Sorry Vitalie.

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/OfficeDev/office-ui-fabric-react/pull/8964)

### CODE REVIEW

**Comment by bigbadcapers** [comment]
@Jahnp you may want eyes on this as well

**Comment by bigbadcapers** [comment]
I'm having trouble double-checking that the image references in the markdowns point to valid (non-404) images in the CDN. @Jahnp @ecraig12345 are the URLs in the cdn case sensitive or something?

File: apps/fabric-website/src/pages/Controls/ChipPage/docs/android/ChipOverview.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+Chips are compact representations of entities (most commonly, people) that can be typed in, deleted or dragged easily. The most common example can be seen in the To field of a new email in Outlook Mobile.
+
+The chip field control handles keyboard input, wrapping and truncation automatically.
+
+|People Picker |Persona Chips |
+|---|---|
+|<img src="https://static2.sharepointonline.com/files/fabric/fabric-website/images/controls/android/Persona/PeoplePickerView.png" alt="People Picker in Android" style="width: 50%;" /> |<img src="https://static2.sharepointonline.com/files/fabric/fabric-website/images/controls/android/Persona/PersonaChipView.png" alt="Persona Chips in Android" style="width: 50%;" /> |
```
Comment (Jahnp) [inline apps/fabric-website/src/pages/Controls/ChipPage/docs/android/ChipOverview.md]: Per your top-level comment, unfortunately, the images are all lower-case :( It's a CDN setting. So the following URL will work: https://static2.sharepointonline.com/files/fabric/fabric-website/images/controls/android/persona/peoplepickerview.png

But the one in your code won't right now. So sorry for the inconvenience!

**Comment by bigbadcapers** [comment]
ah I didn’t expect that. it’s case insensitive for filetype icons. fixing now. #learningMyVSCodeShortcutsLikeABoss

On May 4, 2019, at 11:31 AM, Peter Jahn <notifications@github.com<mailto:notifications@github.com>> wrote:


@Jahnp commented on this pull request.

________________________________

In apps/fabric-website/src/pages/Controls/ChipPage/docs/android/ChipOverview.md<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FOfficeDev%2Foffice-ui-fabric-react%2Fpull%2F8964%23discussion_r280991259&data=02%7C01%7Ccaperez%40microsoft.com%7Ce903662bde894ee1087708d6d0bec454%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C636925915087577283&sdata=rELm6eYj6he4gCOpjdK%2FNaPIO7HpZKj5emskVDiXq2M%3D&reserved=0>:

> @@ -0,0 +1,7 @@
+Chips are compact representations of entities (most commonly, people) that can be typed in, deleted or dragged easily. The most common example can be seen in the To field of a new email in Outlook Mobile.
+
+The chip field control handles keyboard input, wrapping and truncation automatically.
+
+|People Picker |Persona Chips |
+|---|---|
+|People Picker in Android |Persona Chips in Android |


Per your top-level comment, unfortunately, the images are all lower-case :( It's a CDN setting. So the following URL will work: https://static2.sharepointonline.com/files/fabric/fabric-website/images/controls/android/persona/peoplepickerview.png<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fstatic2.sharepointonline.com%2Ffiles%2Ffabric%2Ffabric-website%2Fimages%2Fcontrols%2Fandroid%2Fpersona%2Fpeoplepickerview.png&data=02%7C01%7Ccaperez%40microsoft.com%7Ce903662bde894ee1087708d6d0bec454%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C636925915087577283&sdata=HwoH3CkLccjY%2B0bCWtKMaVgkWtQvg4d%2FghKBvc15ijE%3D&reserved=0>

But the one in your code won't right now. So sorry for the inconvenience!

**Comment by bigbadcapers** [comment]
@Jahnp should i be worried?
image

**Comment by Jahnp** [comment]
@bigbadcapers looks like you have some merge conflicts. I'll ping you offline!

**Comment by Jahnp** [comment]
@bigbadcapers I'm going to close this one down for cleanliness. I've opened #8965 as the substitute. Can you take a look there?

**Pull request closed without merging**